eng/SourceBuild.props

```diff
@@ -8,9 +8,11 @@
     <SourceBuildTargetFrameworkFilter>netstandard2.0%3bnetstandard2.1%3bnetcoreapp2.1%3bnetcoreapp3.1%3bnet5.0%3bnet6.0%3bnet7.0%3bnet8.0%3bnet9.0</SourceBuildTargetFrameworkFilter>
   </PropertyGroup>
 
+  <!-- Only run this target in source-build only mode. -->
```

Comment:
This comments surfaces a UX issue we have with the name of this file.  It gives the impression this file is only in scope of a source-build.

@mmitche - Do we have an issue to rename SourceBuild.props?

Reply:
https://github.com/dotnet/source-build/issues/3963

Reply:
Thanks.  I am going to update that issue to have a running list of known items to rename.